src/sdg/api/permissions.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework.permissions import SAFE_METHODS, BasePermission
+from vng_api_common.permissions import bypass_permissions
+
+
+class LocationPermission(BasePermission):
+    def has_permission(self, request, view):
+        if bypass_permissions(request):
+            return True
+
+        if request.method in SAFE_METHODS:
+            return True
+
+        if not request.auth:
+            return False
+
+        if view.action == "create":
+            organisatie = view.get_organisatie(request, view)
+
+            if organisatie.get("owms_pref_label"):
+                if (
+                    request.auth.lokale_overheid.organisatie.owms_pref_label
+                    != organisatie.get("owms_pref_label")
```

Comment:
With the above mentioned change of the Token auth, this should probably read something like (pseudo):

```suggestion
                    organisatie.get("owms_pref_label") not in request.auth.lokale_overheid_set.values_list("owms_pref_label", flat=True)
```